source/codegen/metadata/nidcpower/CHANGES.md

```diff
@@ -54,6 +52,29 @@ Following enums were defined :
     - NIDCPOWER_VAL_POWER_ALLOCATION_MODE_AUTOMATIC to NIDCPOWER_VAL_AUTOMATIC
     - NIDCPOWER_VAL_POWER_ALLOCATION_MODE_MANUAL to NIDCPOWER_VAL_MANUAL
 
+The following enums were removed :
```

Comment:
By looking at the lines removed above, seems like these enums were not part of nimi-python metadata. These were added by us. So just removing lines above should be sufficient, we don't need to document these as "removed enums".

Reply:
Done.